Automattic/wp-calypso
# Add new nesletter processing background

## Proposed Changes

Add new processing background for newsletter according to the design

<img width="656" alt="image" src="https://user-images.githubusercontent.com/2653810/187432037-cea6885b-ec6c-4ce8-a6b3-17130e0b5297.png">


## Testing Instructions

- Pull this branch and run yarn start
- Navigate to http://calypso.localhost:3000/setup/intro?flow=newsletter and go further until after the Domain step.
- On the process screen, the design should match.
